sdk/core/Azure.Core/samples/Serialization.md

```diff
@@ -1,34 +1,48 @@
-# Azure.Core public serialization samples
+# Azure.Core Public Serialization Guide
 
-## Using explicit cast
+The latest updates to the `Azure.Core` library have simplified serializing and deserializing of all public Azure models. With the addiion of the `ModelSerializer` class, we provide a public interface that exposes the previously internal serialization code so customers can access it without needing to use reflection or write their own translation. The recommended serialization approach is to use the static `ModelSerializer` class. The `ModelSerializer` class also has `ModelSerializerOptions` which allows the user to set the serialization `ModelSerializerFormat` type (`Json` or `Wire`) and provide custom Serializer types for specific models.
```

Comment:
```suggestion
The `Azure.Core` library supports serialization and deserialization of most client library model types using the static `ModelSerializer` class.  Using `ModelSerializerOptions` with the serializer allows you control over formatting and to override serialization behavior.  
```
- As a sample, we should be looking at this as a reference and not a point-in-time announcement of changes.  I'd suggest avoiding phrases like "the latest updates" and "with the addition" in favor of just discussing what the feature is and does. 

- I'd stay away from superlatives.  We can't guarantee ALL models now and forever.

- I'd suggest adding a dedicated section later in the sample that discusses the most useful options and what they mean.  Best to do that after the basic "how do I use this in the default case" sections.

Reply:
+1 re: point in time.